pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/AvoidUsingHardCodedIPRule.java

```diff
@@ -35,10 +39,10 @@
     }
 
 
-    public static final EnumeratedMultiProperty<String> CHECK_ADDRESS_TYPES_DESCRIPTOR = new EnumeratedMultiProperty<>(
-        "checkAddressTypes", "Check for IP address types.", ADDRESSES_TO_CHECK,
-        Arrays.asList(IPV4, IPV6, IPV4_MAPPED_IPV6),
-        String.class, 2.0f);
+    private static final PropertyDescriptor<List<String>> CHECK_ADDRESS_TYPES_DESCRIPTOR =
```

Comment:
I hope, it's ok to change this from public to private and change the type? I guess, nobody anyway directly used this property programmatically...

Reply:
I think, we should keep it public...